README.md

```diff
@@ -13,33 +12,42 @@ project reserves the right to withdraw and abandon the development of this proje
 if it is not sustainable.
 
 ## Getting started
+
+This section explains how to run Spark Connect Go locally.
+
+Step 1: Install Golang: https://go.dev/doc/install.
+
+Step 2: Ensure you have installed `buf CLI` installed, [more info here](https://buf.build/docs/installation/)
+
+Step 3: Run the following commands to setup the Spark Connect client.
+
 ```
 git clone https://github.com/apache/spark-connect-go.git
 git submodule update --init --recursive
 
 make gen && make test
 ```
-> Ensure you have installed `buf CLI`; [more info](https://buf.build/docs/installation/)
 
-## How to write Spark Connect Go Application in your own project
+Step 4: Setup the Spark Driver on localhost.
 
-See [Quick Start Guide](quick-start.md)
+1. [Download Spark distribution](https://spark.apache.org/downloads.html) (3.4.0+), unzip the package.
 
-## Spark Connect Go Application Example
+2. Start the Spark Connect server with the following command:
```

Comment:
```suggestion
2. Start the Spark Connect server with the following command (make sure to use a package version that matches your Spark distribution):
```

Reply:
@grundprinzip - updated this language, good suggestion.